src/Webserver/Configuration.php

```diff
@@ -7,8 +7,10 @@
     const DEFAULT_PORT = 8000;
     const DEFAULT_HOST = 'localhost';
     const DEFAULT_DOCROOT = '/web';
+    const DEFAULT_ROUTER = 'index.php';
 
     public function getPort();
     public function getHost();
     public function getDocroot();
+    public function getRouter();
 }
```

Comment:
I still have to figure out why I need to add two lines at the end instead of one though. PHPStorm just adds a new line but it doesn't seem to be enough for github. Meh.